docs/roles/developer/contracts/api.md

```diff
@@ -0,0 +1,15 @@
+---
+id: api 
+title: API
+sidebar_label: API
+---
+
+The generic solution to invoking smart contracts on the NEAR Platform is our RPC interface. This method is used by `near-api-js`. Our [RPC interface](/docs/api/rpc) uses Standard JSON RPC 2.0 across the board.
+
+For `view` methods:
```

Comment:
It is worth linking readers to the page where we describe the difference between view and change methods

Reply:
> This is a much better structure! Thank you for this revamp!
thank you soooo much @frol! really appreciate it :)

Will link the readers to the view / change method information as well.